internal/stack/stacks_test.go

```diff
@@ -134,6 +163,43 @@ func TestAllLargeStack(t *testing.T) {
 	close(done)
 }
 
+func TestParseFuncName(t *testing.T) {
+	tests := []struct {
+		name string
+		give string
+		want string
+	}{
+		{
+			name: "function",
+			give: "example.com/foo/bar.baz()",
+			want: "example.com/foo/bar.baz",
+		},
+		{
+			name: "method",
+			give: "example.com/foo/bar.(*baz).qux()",
+			want: "example.com/foo/bar.(*baz).qux",
+		},
+		{
+			name: "created by", // Go 1.20
+			give: "created by example.com/foo/bar.baz",
+			want: "example.com/foo/bar.baz",
+		},
+		{
+			name: "created by/in goroutine", // Go 1.21
```

Comment:
should we add some testdata of real stack traces from go 1.20 / 1.21 and ensure they parse correctly too?